examples/b2c/checkout/integrations/classes/B2BDeliverySampleTest.cls

```diff
@@ -3,7 +3,7 @@ private class B2BDeliverySampleTest {
     @testSetup static void setup() {
         Account testAccount = new Account(Name='TestAccount');
         insert testAccount;
-        WebStore testWebStore = new WebStore(Name='TestWebStore', DefaultLanguage='en_US');
+        WebStore testWebStore = new WebStore(Name='TestWebStore', SupportedLanguages='en_US', DefaultLanguage='en_US');
```

Comment:
This is just a test. And no language functionality is being tested. "Supported Languages" was added because these tests started failing after we made the translation workbench changes.